pyvrp/Population.py

```diff
@@ -146,6 +146,12 @@ def purge(self):
             # population, and remove it.
             self.remove(max(self, key=lambda item: item.fitness).individual)
 
+    def restart(self):
```

Comment:
Suggestion: we can recreate the subpops in `Population.restart`, which makes this function unnecessary.